README.rst

```diff
@@ -118,6 +121,80 @@ Short instructions
 
    -  Admin - Username: admin \| Password: admin
 
+2. Yaksh Installation as a django-app
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+1. pip install git+https://github.com/FOSSEE/online_test
+
+2. Add following apps to the INSTALLED_APPS in the project settings:
+    - yaksh
+    - taggit
+    - grades
+    - social.apps.django_app.default
+
+3. On terminal, run the migrate command:
+
+   ::
+
+       $ python manage.py migrate
+
+4. Add following in the project urls.py
+
+   ::
+
+        from django.conf.urls import url, include
+
+        urlpatterns += [
+            url(r'^exam/', include('yaksh.urls', namespace='yaksh', app_name='yaksh')),
+            url(r'^exam/reset/', include('yaksh.urls_password_reset')),
+            url(r'^', include('social.apps.django_app.urls', namespace='social')),
+            url(r'^grades/', include('grades.urls', namespace='grades', app_name='grades')),
+        ]
+
+Note, here there should be static url for Media files,
+this is necessary for file upload/download to work. Following is for reference:
+
+   ::
+
+        In project settings file
+
+        MEDIA_URL = '/'
+        MEDIA_ROOT = BASE_DIR
+
+        In project root url
+
+        from django.conf.urls.static import static
+        from django.conf import settings
+
+        urlpatterns += static(settings.media_url, document_root=settings.media_root)
```

Comment:
This should be `MEDIA_ROOT` and `MEDIA_URL`. Also can't we just store all the uploaded files in `yaksh_data` folder with a directory `data`. Currently, it stores all the uploaded files in the project directory which I feel becomes too clumsy if lots of files are uploaded

Reply:
This an app, so mostly it will be installed in site packages. The project settings will decide where to store media files.